tests/test_breakpoint.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 
-import aesara
+from aesara import function, grad
```

Comment:
Lik in the comment above, this import still goes through the main `__init__`, because `function, grad` aren't modules.